### PR TITLE
add cookie consent banner and functionality for it

### PR DESCRIPTION
**Pull request by fuskovic**
This PR does the following 
- adds a cookie consent banner
- script adds cookie to user browser on load
- script checks for cookie and does not display banner if cookie is already present

### CODE REVIEW

**Comment by fuskovic** [comment]
Pending staging approval

**Comment by fuskovic** [comment]
changed script to only add cookie on banner close per Andi

**Review by VinozzZ — CHANGES_REQUESTED** [review]
Left some comments

File: layouts/partials/site-scripts.html [hunk 1]
```diff
@@ -4,4 +4,53 @@
...
+    function select(e){
+        var currentTab = e.parentNode;
+        var choices = currentTab.parentNode.children;
+        for(var i = 0; i < choices.length; i++){
+            choices[i].children[0].classList.remove("active");
+            choices[i].style.borderLeft = "2px solid #DCDCDC";
+            choices[i].style.opacity = 0.5 ;
+            choices[i].style.fontWeight = 500;
+        }
+        currentTab.style.borderLeft = "2px solid #2582FF";
```
[the remark this answers is not in the document]
Comment (fuskovic) [reply layouts/partials/site-scripts.html]: onClick for the use-cases partial. It's not related to this PR I just moved it into site-scripts to refactor it some.

File: layouts/partials/site-scripts.html [hunk 2]
```diff
@@ -4,4 +4,53 @@
 <script type='text/javascript' src='//s3.amazonaws.com/downloads.mailchimp.com/js/mc-validate.js'></script><script type='text/javascript'>(function($) {window.fnames = new Array(); window.ftypes = new Array();fnames[0]='EMAIL';ftypes[0]='email';fnames[1]='FNAME';ftypes[1]='text';fnames[2]='LNAME';ftypes[2]='text';fnames[3]='ADDRESS';ftypes[3]='address';fnames[4]='PHONE';ftypes[4]='phone';}(jQuery));var $mcj = jQuery.noConflict(true);</script>
 <script type='text/javascript' src='//s3.amazonaws.com/downloads.mailchimp.com/js/mc-validate.js'></script><script type='text/javascript'>(function($) {window.fnames = new Array(); window.ftypes = new Array();fnames[0]='EMAIL';ftypes[0]='email';}(jQuery));var $mcj = jQuery.noConflict(true);</script>
 <script type='text/javascript' src='//s3.amazonaws.com/downloads.mailchimp.com/js/mc-validate.js'></script><script type='text/javascript'>(function($) {window.fnames = new Array(); window.ftypes = new Array();fnames[0]='EMAIL';ftypes[0]='email';fnames[1]='MMERGE1';ftypes[1]='text';fnames[2]='MMERGE2';ftypes[2]='text';}(jQuery));var $mcj = jQuery.noConflict(true);</script>
+<script>
+    function select(e){
```
[the remark this answers is not in the document]
Comment (fuskovic) [reply layouts/partials/site-scripts.html]: onClick for the use-cases partial. It's not related to this PR I just moved it into site-scripts to refactor it some.

File: assets/scss/_base.scss [hunk 3]
```diff
@@ -49,3 +49,38 @@ footer {
   position: fixed;
   bottom: 0;
 }
+
+.consent-banner{
```
[the remark this answers is not in the document]
Comment (fuskovic) [reply assets/scss/_base.scss]: the banner goes on the whole site.

[the remark this answers is not in the document]
Comment (VinozzZ) [reply assets/scss/_base.scss]: Yes, I meant that the rules are global in _base.scss in terms o f elements. It's for rules that will affect native HTML elements across the whole site. Maybe I'm not explaining it well. Check this article https://matthewelsom.com/blog/simple-scss-playbook.html#base. 
To me, the `.consent-banner` is a component specific rule. it only affects the banner no matter where the banner is used. It doesn't affect other parts of the page. Does that make sense?

File: layouts/partials/site-scripts.html [hunk 4]
```diff
@@ -4,4 +4,53 @@
...
+        var currentTab = e.parentNode;
+        var choices = currentTab.parentNode.children;
+        for(var i = 0; i < choices.length; i++){
+            choices[i].children[0].classList.remove("active");
+            choices[i].style.borderLeft = "2px solid #DCDCDC";
```
[the remark this answers is not in the document]
Comment (fuskovic) [reply layouts/partials/site-scripts.html]: this does not work using classes the way it should due to bootstrap specificity issues - even when you add !important. Also, this is functionality that we decided to take out anyway in the near future with Atikh and is also unrelated to the PR so I will leave it as is until we remove it entirely in the next sprint or two. This is for the use-cases image changing onclick when I did the vanguard update on the homepage.

File: layouts/partials/site-scripts.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (fuskovic) [reply layouts/partials/site-scripts.html]: this does not work using classes the way it should due to bootstrap specificity issues - even when you add !important. Also, this is functionality that we decided to take out anyway in the near future with Atikh and is also unrelated to the PR so I will leave it as is until we remove it entirely in the next sprint or two. This is for the use-cases image changing onclick when I did the vanguard update on the homepage.

File: assets/scss/_base.scss (hunk 3 above)
[the remark this answers is not in the document]
Comment (fuskovic) [reply assets/scss/_base.scss]: It just seems weird to make an entirely new SCSS file for one element. The consent banner is actually applied in the footer partial. By that logic, I should be adding the rules in the _footer.scss file right ?

[the remark this answers is not in the document]
Comment (fuskovic) [reply assets/scss/_base.scss]: Ok I made a new file for this.

File: assets/scss/_base.scss [hunk 5]
```diff
@@ -49,3 +49,38 @@ footer {
...
+  background: #11181D;
+  opacity: 0.97;
+  position: fixed;
+  bottom: 0px;
+  height: 60px;
+  z-index: 99999999999999999;
+
+  &_text {
+    margin-top: 25px;
+    font-family: "Roboto";
```
[the remark this answers is not in the document]
Comment (fuskovic) [reply assets/scss/_base.scss]: done

File: layouts/partials/consentBanner.html [hunk 6]
```diff
@@ -0,0 +1,4 @@
+<div class="container-fluid d-flex justify-content-between consent-banner" id="consent-banner">
+        <p class="consent-banner_text">This site uses cookies. By continuing to use this site you are agreeing to our <a class="consent-banner_text" style="text-decoration:underline;" href="https://storj.io/privacy-policy">use of cookies and privacy policy</a>.</p>
```
[the remark this answers is not in the document]
Comment (fuskovic) [reply layouts/partials/consentBanner.html]: done

**Review by VinozzZ — APPROVED** [review]

**Pull request merged**